app/services/idv/steps/document_capture_step.rb

```diff
@@ -2,18 +2,28 @@ module Idv
   module Steps
     class DocumentCaptureStep < DocAuthBaseStep
       def call
-        response = post_images
-        if response.success?
+        api_upload = flow_session['api_upload']
+        response = (api_upload && api_upload['documents']) || post_form_images
+        handle_response(response)
+      end
+
+      private
+
+      def post_form_images
+        selfie = selfie_image ? selfie_image.read : nil
+        post_images(front_image.read, back_image.read, selfie)
+      end
+
+      def handle_response(response)
+        if response.to_h[:success]
```

Comment:
usually our response objects have `.success?`

```suggestion
        if response.success?
```